packages/react-cosmos-webpack/src/webpack-config.js

```diff
@@ -44,11 +44,13 @@ export default function getWebpackConfig(
     publicPath: '/loader/',
   };
 
-  const loaders = userWebpackConfig.module && userWebpackConfig.module.loaders ?
-    [...userWebpackConfig.module.loaders] : [];
+  // use what user wants
```

Comment:
Can you please improve this comment to explain the reason behind this is to supporting webpack 1 and 2?